science/projects/Group Projects/01-main-content/parser.ipynb

```diff
@@ -0,0 +1,735 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    leaf_nodes = [node for node in 
</pre><p>If possible, please write down what attributes Parser provides us with.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>